### PR TITLE
plaintext-accounting: expand/cleanup

### PR DESCRIPTION
**Pull request by simonmichael**

### CODE REVIEW

File: topics/plaintext-accounting/index.md [hunk 1]
```diff
@@ -7,7 +7,19 @@ url: https://plaintextaccounting.org
...
 
-Plain text accounting is a way of doing bookkeeping and accounting with plain text files
+***Plain text accounting*** is a way of doing bookkeeping and accounting with plain text files
 and scriptable, command-line-friendly software, such as
 [Ledger](https://ledger-cli.org/), [hledger](https://hledger.org/), or
 [Beancount](https://beancount.github.io/).
+Benefits include version control, scriptability, data portability and privacy.
+See https://plaintextaccounting.org and https://wiki.plaintextaccounting.org for more details.
+
+If you have a PTA-ish project on Github, consider adding the `#plaintext-accounting` (*) topic in your repo settings.
```
Comment (ahpook) [inline topics/plaintext-accounting/index.md:17]: Our topics don't use a leading `#` in their names when you add them to a repo.

File: topics/plaintext-accounting/index.md [hunk 2]
```diff
@@ -7,7 +7,19 @@ url: https://plaintextaccounting.org
...
+
+Some related topics:
+[`#ledger-cli`](https://github.com/topics/ledger-cli),
+[`#hledger`](https://github.com/topics/hledger),
+[`#beancount`](https://github.com/topics/beancount).
```
Comment (ahpook) [inline topics/plaintext-accounting/index.md:22]: Same here, no leading `#`

**Review by ahpook — CHANGES_REQUESTED** [review]
Thanks for the PR! Please remove the `#`s, and I'll go ahead and merge it.

**Comment by simonmichael** [comment]
Done

**Review by ahpook — APPROVED** [review]
Looks great, thanks!

File: topics/plaintext-accounting/index.md (hunk 2 above)
Comment (Mralfa68) [reply topics/plaintext-accounting/index.md]: ****

**Comment by simonmichael** [comment]
Thanks!

**Pull request merged**